public/locales/en/modules/media-requests-stats.json

```diff
@@ -5,7 +5,11 @@
     "settings": {
       "title": "Media requests stats",
       "direction": {
-        "label": "Direction of the layout."
+        "label": "Direction of the layout.",
+        "data":{
+          "row": "Horizontal",
+          "column": "Vertical"
```

Comment:
I've seen it translated elsewhere. Put into common.json if it's common.

Reply:
That property translations are used by the widget and so it is not possible to move it to the common.json